### DISCUSSION TITLE
posts/blog-comments-are-here/

### DISCUSSION THREAD

**Discussion by giscus** [amtoine/amtoine.github.io#3] [blog]
# posts/blog-comments-are-here/

Blog comments are here!

https://amtoine.github.io/posts/blog-comments-are-here/

**Comment by amtoine** [comment]
hello world! :yum:

**Reply by amtoine** [reply]
test reply from github.

**Reply by amtoine** [reply]
test reply from amtoine.github.io.

**Reply by cassiofb-dev** [reply]
> [!NOTE]  
> Testing admonitions.
